tools/runners/test262-harness.py

```diff
@@ -518,6 +574,8 @@ def get_negative_type(self):
 
     def get_negative_phase(self):
         negative = self.get_negative()
+        if self.suite.es5_1:
+            return negative
```

Comment:
Cannot be the negative record returned by the ES5.1 parser be made more alike to that of ES.next? This would allow avoiding special casing like this.

Reply:
Yeap, looking for suggestion, maybe have better way to solve this issue

Reply:
fixed